src/main/java/com/unascribed/fabrication/features/FeatureLegacyCommandSyntax.java

```diff
@@ -71,7 +71,8 @@ public void apply() {
 									serverPlayerEntity.addExperience(amount);
 									c.getSource().sendFeedback(new TranslatableText("commands.experience.add.points.success.single", amount, serverPlayerEntity.getDisplayName()), true);
 									return 1;
-								})).then(CommandManager.argument("lvlAmount", StringArgumentType.word()).executes((c) -> {
+								}))
+							.then(CommandManager.argument("lvlAmount", StringArgumentType.word()).executes((c) -> {
```

Comment:
This should be outdented another level; my bad. GitHub's suggestion viewer thing is hard to read with indents this long.